### ISSUE TITLE
Use live data instead of flat files.

### ISSUE THREAD

**Issue by ryayak1460** [Code4HR/local-art-api#23]
- [x] Put existing flat file into OpenStreetMap.

    id => node:id
    title => tag:name
    longitude => node:lon
    latitude => node:lat
    location => ? (may no longer have relevance)
    artists => tag:artist_name
    url => tag:source
    imageurl => tag:website_1
    fullimage => tag:website
    description => tag:note, tag:note_1 if more required.

  - [x] Use query-overpass npm module to load nodes from OSM and return base data in a format the client expects.
  - [ ] Stand up a MongoDB image store with GridFS, using the following structure.
    
    {node, thumbnail, image}

  - [ ] Include instructions on getting the MongoDB instance running and working with the proxy.
  - [ ] Include an images route for both the client and OSM to send users for images and thumbnails.
    
    GET /images/node: returns an image with the appropriate MIME type.
    PUT /images: creates a thumbnail and places the image in the GridFS store.

  - [ ] Get OAuth working for the proxy using the iArtNorfolk OSM account.
  - [ ] Map Ember.js REST API calls to OSM API calls.

**Comment by wbprice** [comment]
Exhibits route is implemented here:
```
http://localartapi-code4hr.rhcloud.com/exhibits
http://localartapi-code4hr.rhcloud.com/exhibits?bbox=36.75,-76.44,36.98,-76.13
```

**Comment by ryayak1460** [comment]
@wbprice, does the bounding box work here?

**Comment by wbprice** [comment]
Yes.  If you don't send a bounding box to the /exhibits route it will assume ```bbox=36.75,-76.44,36.98,-76.13```, but you specify a larger or smaller box as you need to.

**Comment by ryayak1460** [comment]
Wow, awesome, man!

Also, I just went through and updated the descriptions and fullimages in each OSM node, then committed the OpenShift URL to our repo.  Once we can correct some style things with images on the front page, we should push that to production.

**Comment by stanzheng** [comment]
i think this has been implemented

**Comment by ryayak1460** [comment]
We still have 5/7 subtasks that we have not implemented, this is not closed yet.
